app/models/applicant.rb

```diff
@@ -0,0 +1,18 @@
+require 'date'
+
+class Applicant < ApplicationRecord
+
+  has_one :legal_aid_application
+
+  validates :name, :date_of_birth, presence: true
+
+  validate :validate_date_of_birth
+
+  private
+
+  def validate_date_of_birth
+    if date_of_birth.present? && (date_of_birth < Date.new(1900, 01, 01) || date_of_birth > Date.today)
+       errors.add(:date_of_birth, "Date of birth is not valid")
```

Comment:
**[not a blocker]** Going forward consider using error codes instead of strings, making use of [Rails i18n](https://guides.rubyonrails.org/i18n.html#error-message-scopes) so you can define the error messages in the appropriate locales (normally set in `config/locales/en.yml` for en-GB)